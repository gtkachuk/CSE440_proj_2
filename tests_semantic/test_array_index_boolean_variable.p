program testArrayIndexBooleanVariable;

class testArrayIndexBooleanVariable

BEGIN

   VAR 
      rows    : ARRAY[0 .. 9] OF boolean;
      tests   : boolean;
      someVar : boolean;

FUNCTION testArrayIndexBooleanVariable;
BEGIN	   
   tests := rows[someVar]
END

END
.

