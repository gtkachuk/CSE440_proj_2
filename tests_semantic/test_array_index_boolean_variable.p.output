ERROR:14:Array index not an integer for variable "rows"
Errors detected. Exiting.
