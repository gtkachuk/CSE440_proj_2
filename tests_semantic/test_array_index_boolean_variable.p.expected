ERROR:15:Array index not an integer for variable "rows"
Errors detected. Exiting.
